tests/appsec/api_security/test_apisec_sampling.py

```diff
@@ -37,6 +38,7 @@ def setup_sampling_rate(self):
             for _ in range(self.N ** 2)
         ]
 
+    @irrelevant(True, reason="RFC is deprecated by a newer RFC. New tests will be implemented")
```

Comment:
@christophe-papazian i see you've made a commit `do not touch other libs` but doesn't this affect other libs ?